source/features/show-pr-file-state.tsx

```diff
@@ -0,0 +1,51 @@
+import select from 'select-dom';
+import {observe} from 'selector-observer';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {observeOneMutation} from '../helpers/simplified-element-observer';
+
+function loadJumpList(jumpList: Element): void {
+	jumpList.parentElement!.dispatchEvent(new MouseEvent('mouseover'));
+}
+
+async function init(): Promise<false | void> {
+	const jumpList = await elementReady('details.toc-select details-menu')!;
+	loadJumpList(jumpList!);
+	setTimeout(loadJumpList, 1000, jumpList!);
```

Comment:
Don't forget comments, this doesn't make sense otherwise 

https://github.com/sindresorhus/refined-github/blob/e2932e733196bc09e9842262575a00732bcbe54d/source/features/reactions-avatars.tsx#L60-L63

Reply:
I know. I was not sure if you even wanted to do this way at the end. I am not done with the PR yet